src/lib/process/api.ts

```diff
@@ -0,0 +1,97 @@
+import { Restrictions } from '../types/restrictions.js'
+import { Options } from '../types/options.js'
```

Comment:
This is something we should tell config to automatically import all the types from `/types` folder so we can just use them across the process wiithout needing an import